src/WebJobs.Extensions.NotificationHubs/NotificationHubAttribute.cs

```diff
@@ -39,12 +39,14 @@ public sealed class NotificationHubAttribute : Attribute
         /// Optional. A string value indicating the app setting to use as the Notification Hubs connection
         /// string, if different than the one specified in the <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
         public string ConnectionStringSetting { get; set; }
 
         /// <summary>
         /// Optional. The Notification Hub Name to use, if different than the one specified in the
         /// <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
```

Comment:
Depends -- is this filled in by the picker? If so, it should be AllowTokens = false. This means it will always be an app setting and no direct strings are allowed.
@pragnagopa -- Is that the intent?


---
In reply to: [70489560](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/120#discussion_r70489560) [](ancestors = 70489560)